zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/NetworkUtil.java

```diff
@@ -22,77 +22,14 @@
 import java.io.IOException;
 import java.net.ConnectException;
 import java.net.InetAddress;
-import java.net.NetworkInterface;
 import java.net.Socket;
 import java.net.SocketException;
 import java.net.UnknownHostException;
-import java.util.Enumeration;
-import java.util.HashSet;
-import java.util.Set;
 
 public class NetworkUtil {
 
     public final static String OVERWRITE_HOSTNAME_SYSTEM_PROPERTY = "zkclient.hostname.overwritten";
 
-    public static String[] getLocalHostNames() {
```

Comment:
Delete public API is not allowed in open source project. Even if we are not using it in another of our code, it will be dangerous that open source users may use it.

Please keep it here, if no place to use this API in our code, you can annotate with @Deprecated.

Reply:
Ohh ok. Gotcha! I won't mark it as deprecated since it's utility class and it delivers on the functionality. I will just revert the file to it's original state.